frontend/lib/src/components/widgets/Selectbox/Selectbox.test.tsx

```diff
@@ -153,4 +153,41 @@ describe("Selectbox widget", () => {
       undefined
     )
   })
+
+  it("maintains scroll position when reopening dropdown", () => {
+    const props = getProps({
+      options: Array.from({ length: 100 }, (_, i) => `Option ${i}`),
+    })
+    vi.spyOn(Utils, "convertRemToPx").mockImplementation(mockConvertRemToPx)
+
+    render(<Selectbox {...props} />)
+    const selectbox = screen.getByRole("combobox")
+
+    // Open dropdown
+    // TODO: Utilize user-event instead of fireEvent
+    // eslint-disable-next-line testing-library/prefer-user-event
+    fireEvent.click(selectbox)
+
+    // Get dropdown content and scroll
+    const dropdown = screen.getByTestId("stSelectboxVirtualDropdown")
+    act(() => {
+      // Simulate scrolling down
+      const scrollEvent = new Event("scroll", { bubbles: true })
+      Object.defineProperty(dropdown, "scrollTop", { value: 500 })
+      dropdown.dispatchEvent(scrollEvent)
+    })
+
+    // Close dropdown
+    // TODO: Utilize user-event instead of fireEvent
+    // eslint-disable-next-line testing-library/prefer-user-event
+    fireEvent.keyDown(selectbox, { key: "Escape" })
```

Comment:
nit: did you try to use `user-event` here? Did it cause some issues?

Reply:
Ah I thought I did, but apparently I just used `fireEvent` because it was used in the other test above. I tried it out and it work, so I changed this test to `userEvent`. Unfortunately, I couldn't get the other test above with `fireEvent` to work with `userEvent`, so I'm leaving that as is for now.